src/ml-search.service.js

```diff
@@ -629,9 +629,9 @@
      */
     getCombinedQuery: function getCombinedQuery(includeOptions) {
       var d = $q.defer(),
-          combined = {
-            search: { query: this.getQuery() }
-          };
+        combined = {
+          search: { query: this.getQuery() }
+        };
 
```

Comment:
Can you suggest a way to make jshint not complain about this in my TextMate editor?